superbit_lensing/metacalibration/ngmix_fit_superbit3.py

```diff
@@ -238,7 +238,8 @@ def _get_source_observations(self,source_id = None,psf_noise = 1e-6):
             #     - std_dev = sqrt(bkg) = 5.3
             #     - sky_sigma = std_dev**2 = 25.1
 
-            sky_sigma = (4.7)**2
+            sky_sigma = 4.77 #SIGMA = 4.7ish for Blue, 4.7**2 for shape
```

Comment:
True. Any ideas for how we might do that?

Reply:
In #37 I think I mentioned that we could save this to the SCI image headers since that will probably happen with real data. But we could also just pass in the used galsim config for now, since actually making this data-ready would take many more changes (like a separate `sky_sigma` for each image, for example)